install

```diff
@@ -5,8 +5,15 @@ version=0.9.7
 cd $(dirname $BASH_SOURCE)
 fzf_base=$(pwd)
 
+# If stdin is a tty, we are "interactive".
+[ -t 0 ] && interactive=yes
```

Comment:
This is actually posix compatible in case anyone is curious. http://stackoverflow.com/a/911213/152142